tests/test_utils.py

```diff
@@ -1,31 +1,36 @@
+# Builtin libs
 import os
-from collections import OrderedDict, namedtuple
+import re
 import glob
-
 import traceback
+import warnings
+from collections import OrderedDict, namedtuple
+import multiprocessing as mp
 
+# Computation libs
 import numpy as np
 import pandas as pd
 import xarray as xr
-import glob
-import re
+
+# Plottling libs
 import matplotlib
-matplotlib.use('Agg')
 import matplotlib.pyplot as plt
-import warnings
 import seaborn as sns
-import multiprocessing as mp
 
+# Tools from tonic
 from tonic.models.vic.vic import (VICRuntimeError,
                                   default_vic_valgrind_error_code)
 from tonic.testing import check_completed, check_for_nans, VICTestError
 
+matplotlib.use('Agg')
+
```

Comment:
From the pep8 and an operational perspective it is best to do all the imports before applying settings.